src/data_storage/container_storage.ts

```diff
@@ -52,6 +53,14 @@ export default class ContainerStorage extends PostgresStorage{
                               return
                           }
 
+                          for(const i in cs) {
+                            // create graph
+                            GraphStorage.Instance.Create(cs[i].id!, userID).then((result) => {
```

Comment:
we need to log errors in graph storage creation failure so the user knows, we can't return the error because technically the container was created and they'll need the ID, but we still need to let a user know if the graph failed